### PR TITLE
Can pick up mobs in closets

### PR DESCRIPTION
**Pull request by Robertha89**
Not sure why this was changed at all, as this allowed ripley to act as an improvised Odysseus if it got a clamp. Among other things it also allows collecting mining mobs for later use in an easy manner.

### CODE REVIEW

**Comment by tigercat2000** [comment]
> Not sure why this was changed at all

Oh my fucking god, Robertha. This hasn't been touched or changed in 3 years.

**Comment by Robertha89** [comment]
2 years, 2 months and 2 days ago actually... EITHER WAY. it should change anyway, it was a great mechanic.

**Comment by tigercat2000** [comment]
It really shouldn't, or you get people being dicks and trapping people, or people using RIPLEY's as infinite storage. I know, technically it's already achievable by exploit, but that doesn't really make a difference.

**Comment by Robertha89** [comment]
technically the infinity storage is not a bug, and you can already store infinity anything, the only thing this allows is storing mobs, and you can also be a dick and trap people in an odysseus.

**Comment by Spacemanspark** [comment]
Yay.
:+1:

**Comment by Earthdivine** [comment]
On one hand, I can think of many ways to exploit this if It came back , and letting you move massive amount of mining mobs would make traitor miners super powerful.

On the other, you can't save any miners without leaving your Riply behind, and this all your work, and having to walk all the way back. If I wasn't so nice I would legit leaving miners to die because it's so much of a deferment to miners to stop and help someone.

Dunno how I feel about this.

**Comment by Robertha89** [comment]
Traitor miners are already superpowerfull, and more often then not, you already are mining by hand, and reck anything in your path alone, and by the 20 minute mark, if you got 10 of each material, and ask science to upgrade the ore demption machine to the second highest level, it will probably give you around 4-8k points, and from there you git your stuff, and mine more. the only difference is it makes it easier to move several people as a miner, it makes it easier to move hivelords, goliaths and lastly basilisk back to mining if you have a ripley to begin with, it's not faster, it's not less work, it's just simple, and if you got a satchel of holding, you do not need the ore crates, you can just fill a closet with mobs as you kill them, but... Unlike with many other jobs, this require quite a lot of effort, with quite a lot of risk to do, but greater the risk, greater the reward, and yes, I would love to see a traitor miner unleash 7-14 hostile goliaths upon the station, or just the shuttle for that escape alone objective, since if ya aint wearing maxed mining hardsuit, or the riot suit, a goliath will break in one hit your hands.

Tl;DR

This allows miners to use ripliey properly, and having to leave, drag body back, get ripley, go back get ripley is simply a waste of time. Which would not be a problem, cept for the fact that rounds lasts an average 2 hours, so wasting 20 minutes of time to save a life or 8 just ends up taking time away from you to do what you wanted to do in the beginning, to mine. 

Also, nothing scares the station more then leaving living goliaths in maint, friendly or not.

**Comment by Fox-McCloud** [comment]
I don't really have a problem with this---this is something Bay implemented for some reason a long time ago--not sure why, to be honest.

**Comment by Fox-McCloud** [comment]
@Markolie and @TheDZD

**Comment by TheDZD** [comment]
I don't really think that Ripleys should be grabbing and infinitely loading mobs inside of their storage, that is kind of the Odysseus' thing, and even then they're limited to 3 people at most.

**Comment by Robertha89** [comment]
Can't load mobs directly, need to put them in closets first.

**Comment by TheDZD** [comment]
Oh, seems that I may not quite have been awake when I wrote that. The only issues I could really see with this being abused would be people being welded into a locker and then picked up in a Ripley. You might want to test whether or not resisting out of a welded locker works after a Ripley has picked it up.

**Comment by Robertha89** [comment]
You can break out of a welded locker, so this is ready to merge.
